documentation/src/main/asciidoc/topics/code_examples/AnalyzerDefinitions.java

```diff
@@ -0,0 +1,7 @@
+@ProtoDoc("@Field(store = Store.YES, analyze = Analyze.YES, analyzer = @Analyzer(definition = \"keyword\"))")
```

Comment:
@anistor @gustavonalle any change to move forward for 13~14 my annotations request? 🥺

Reply:
I am not looking at it for now, is there a JIRA?

Reply:
https://issues.redhat.com/browse/ISPN-13133